app/indexers/curation_concerns/collection_indexer.rb

```diff
@@ -10,6 +10,11 @@ def generate_solr_document
         # Index the size of the collection in bytes
         solr_doc[Solrizer.solr_name(:bytes, STORED_LONG)] = object.bytes
         solr_doc['thumbnail_path_ss'] = thumbnail_path
+
+        object.in_collections.each do |col|
+          solr_doc['member_of_collection_ids_ssim'] = col.id
```

Comment:
No, this should be using `<<` instead of `=` to allow the object to be in multiple collections — I'll update that.